examples/predictors/lstm/Example-LSTM-Predictor.ipynb

```diff
@@ -289,7 +279,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "predictor = XPrizePredictor(None, DATA_FILE, CUTOFF_DATE)"
+    "predictor = XPrizePredictor(None, DATA_FILE)"
```

Comment:
This is the main change: the `XPrizePredictor` constructor doesn't need a cutoff_date anymore